pkg/operator/staticpod/controller/guard/guard_controller.go

```diff
@@ -201,6 +203,13 @@ func (c *GuardController) sync(ctx context.Context, syncCtx factory.SyncContext)
 				break
 			}
 		}
+		if !found {
+			_, _, err = resourceapply.ApplyPodDisruptionBudget(ctx, c.pdbGetter, syncCtx.Recorder(), pdb)
+			if err != nil {
+				klog.Errorf("Unable to apply PodDisruptionBudget changes: %v", err)
```

Comment:
I think that the errors returned from the sync method are reported by the base controller.

Reply:
The base controller does not log the line number from the guard_controller.go which is useful for quick location of origin of the error message.

Reply:
how about `return fmt.Errorf("Unable to apply PodDisruptionBudget changes: %v", err)` ?

Reply:
Improving the error message sounds good. Though, I would like to keep the `klog.Errof` there so it's easier to locate the line number in the file.

Reply:
fine by me.

Reply:
Done